src/schema/v2/me/conversation/message.ts

```diff
@@ -86,7 +91,24 @@ export const MessageType = new GraphQLObjectType<any, ResolverContext>({
       description:
         "Unaltered text if possible, otherwise `body`: a parsed/sanitized version from Sendgrid.",
       type: GraphQLString,
-      resolve: ({ body, original_text }) => {
+      resolve: ({
+        body,
+        original_text,
+        conversation_from_name,
+        conversation_initial_message,
+        is_first_message,
+      }) => {
+        if (is_first_message) {
+          if (!conversation_initial_message) {
+            throw new Error(
+              "This is the first message, but initial_message not provided"
+            )
+          }
+          const parts = conversation_initial_message.split(
+            "Message from " + conversation_from_name + ":\n\n"
```

Comment:
Was this "Message from..." injected from somewhere?

Reply:
I pulled it over from the `initial_message` field already present in metaphysics. Not sure though.